packages/watchguard_firebox/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,4720 @@
+          outcome: unknown
+          log_type: alarm
+        '3000-0158':
+          category:
+            - intrusion_detection
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+        '3000-0159':
+          category:
+            - intrusion_detection
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+        '3000-0160':
+          category:
+            - intrusion_detection
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+        '3000-0161':
+          category:
+            - intrusion_detection
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+        '3000-0162':
+          category:
+            - intrusion_detection
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+        '3000-0163':
+          category:
+            - intrusion_detection
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+        '3000-0164':
+          category:
+            - intrusion_detection
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+        '3000-0165':
+          category:
+            - intrusion_detection
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+        '3000-0166':
+          category:
+            - intrusion_detection
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+        '3000-0167':
+          category:
+            - network
+          type:
+            - allowed
+          outcome: unknown
+          log_type: alarm
+        '3000-0168':
+          category:
+            - network
+          type:
+            - denied
+          outcome: unknown
+          log_type: alarm
+        '3000-0169':
+          category:
+            - intrusion_detection
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+        '3000-0170':
+          category:
+            - network
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+        '3000-0171':
+          category:
+            - network
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+        '3000-0172':
+          category:
+            - network
+          type:
+            - denied
+          outcome: unknown
+          log_type: alarm
+        '020B-0001':
+          category:
+            - network
+          type:
+            - info
+          outcome: unknown
+          log_type: alarm
+      source: |-
+        if (ctx.watchguard_firebox?.log?.msg_id == null || params.get(ctx.watchguard_firebox.log.msg_id) == null) {
+          return;
+        }
+        def hm = new HashMap(params.get(ctx.watchguard_firebox.log.msg_id));
+        hm.forEach((k, v) -> {
```

```suggestion
        params.get(ctx.watchguard_firebox.log.msg_id).forEach((k, v) -> {
```

This can be simplified down, no need to make a copy of the map in params.